installer/install.iss

```diff
@@ -649,6 +649,10 @@ begin
         Result:='(no output)'
     else
         Result:=Contents;
+    if (Length(Result)>0) and (Result[Length(Result)]=#10) then
+        SetLength(Result,Length(Result)-1);
+    if (Length(Result)>0) and (Result[Length(Result)]=#13) then
+        SetLength(Result,Length(Result)-1);
```

Comment:
Does TrimRight(Result) do this?

Reply:
Almost. It would also trim trailing spaces, which I don't want to do. Also, it would trim more than one trailing newline, which I _also_ don't want to do...

Technically, I only want to trim CR when there was an LF to begin with, but this way the code looks nicer and I don't _really_ care about requiring that LF before the CR...